### PR TITLE
added mcshowcase.net

### PR DESCRIPTION
**Pull request by lukashasmeta**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/mcshowcase/background.png
Comment (DoJapHD) [inline minecraft_servers/mcshowcase/background.png:1]: Compress file

File: minecraft_servers/mcshowcase/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/mcshowcase/background@2x.png:1]: Compress file

File: minecraft_servers/mcshowcase/manifest.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "server_name": "mcshowcase",
+    "nice_name": "MCShowcase.net",
+    "direct_ip": "mcshowcase.net",
+    "social": {
+      "web": "https://mcshowcase.net",
```
Comment (DoJapHD) [inline minecraft_servers/mcshowcase/manifest.json:6]: Maintenance; how long will it take? If it'll take some time, remove and add with another PR

File: minecraft_servers/mcshowcase/manifest.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "web_support": "https://mcshowcase.net",
```
Comment (DoJapHD) [inline minecraft_servers/mcshowcase/manifest.json:7]: Support cannot be the website, has to be FAQ or contact form

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Is the server even released?

**Comment by lukashasmeta** [comment]
> Is the server even released?

No, but the server will be released soon.

**Pull request closed without merging**